vdr/cmd/cmd.go

```diff
@@ -127,14 +127,28 @@ func resolveCmd() *cobra.Command {
 				return fmt.Errorf("failed to resolve DID document: %v", err)
 			}
 
-			for _, o := range []interface{}{doc, meta} {
+			var toPrint []interface{}
+			if !printMetadata && !printDocument {
+				toPrint = append(toPrint, doc, meta)
+			} else {
+				if printDocument {
+					toPrint = append(toPrint, doc)
+				}
+				if printMetadata {
+					toPrint = append(toPrint, meta)
+				}
+			}
+			for _, o := range toPrint {
 				bytes, _ := json.MarshalIndent(o, "", "  ")
 				cmd.Printf("%s\n", string(bytes))
 			}
 
 			return nil
 		},
 	}
+	result.Flags().BoolVar(&printMetadata, "metadata", false, "Pass 'true' to print the metadata.")
```

Comment:
I believe a boolean flag just requires the flag (value is optional)

Reply:
defaults are incorrect. If both are false they will print....

Reply:
> I believe a boolean flag just requires the flag (value is optional)

The var (pointer to `printMetadata`) is optional (you can also use `GetBool`).

> defaults are incorrect. If both are false they will print....

You're correct. I choose it this way because if you don't supply the flags the default (sane) behaviour it will just print both. If you specify one of them, it will only print that one. If you specify both, both are printed. I tried to describe this in the usage but stuff only got complicated, so I guess this is self-explanatory enough. Maybe not to the developer reading this code, but surely to the administrator.

Reply:
I meant if you do `cmd --printMetadata` is equal to `cmd --printMetadata true`.
Maybe change the description to: **pass to only print X**